core/trace/dt_span_exporter.go

```diff
@@ -89,7 +89,7 @@ func (e *dtSpanExporterImpl) export(ctx context.Context, t exportType, spans dtS
 	start := time.Now()
 	// TODO: In order to support large amounts of spans, implement a splitting algorithm
 	// so that we can send spans in batches whose sizes do not exceed cMaxSizeSend.
-	serializedSpans, err := serializeSpans(spans, e.config.Tenant, e.config.AgentId)
+	serializedSpans, err := serializeSpans(spans, e.config.Tenant, e.config.AgentId, e.config.TenantId(), e.config.ClusterId)
```

Comment:
In other languages we have a type like QualifiedTenantId that bundles the two. I recommend also introducing that here, as you may otherwise easily accidentally swap the parameter order.

Reply:
Good point, I'll introduce it. This is too error-prone otherwise.